nullaway/src/main/java/com/uber/nullaway/handlers/ApacheThriftIsSetHandler.java

```diff
@@ -54,7 +55,11 @@
 
   @Override
   public void onMatchTopLevelClass(
-      NullAway analysis, ClassTree tree, VisitorState state, Symbol.ClassSymbol classSymbol) {
+      NullAway analysis,
+      ClassTree tree,
+      VisitorState state,
+      Symbol.ClassSymbol classSymbol,
+      Config config) {
```

Comment:
Don't pass this here, pass it to the handler's constructor in `Handlers.buildDefault(...)`, that's the convention used by existing handlers.

Reply:
Before I go ahead and do a full pass, it looks to me like this remains to be addressed :)

Reply:
Done now 😅